TranslatorReasonerAPI.yaml

```diff
@@ -216,7 +216,13 @@ components:
       properties:
         node_bindings:
           type: object
-          description: List of QNode-KNode bindings.
+          description: >-
+            The dictionary of Input Query Graph to Result Knowledge Graph node
+            bindings where the dictionary keys are the key identifiers of the
+            Query Graph nodes and the associated values of those keys are
+            instances of NodeBinding schema type (see below). This value is an
+            array of NodeBindings since a given query edge may have multiple
```

Comment:
I think "a given query edge" should be "a given query node"

Reply:
Yes, this is a typo that I accidentally carried over from Richard's version.